src/shared/sleep-config.c

```diff
@@ -802,6 +802,12 @@ static bool location_is_resume_device(const HibernateLocation *location, dev_t s
  * Attempt to find the hibernation location by parsing /proc/swaps, /sys/power/resume, and
  * /sys/power/resume_offset.
  *
+ * Beware:
+ *  Never use a device or file as location that hasn't been either explicitly specified by the root user (for example via
```

Comment:
I'd prefer "explicitly by the root user" to be dropped here. It's understood that you need appropriate privileges to specify the resume device, but there are other ways to do this than an action via the root user. For example, this configuration might be written into a config file in the image. (And the image might be created in a different namespace, where the concept of "our" root user is meaningless. It might also be created unprivileged, without a root user being involved at all.)  Thus the addition of "by the root user" and "explicitly" is unnecessary / incorrect.

Please wrap the comment to 109 columns, see CODING_STYLE.

Also, it'd be fine to merge the two commits, they are generally about the same thing.
Also, we don't use signed-off-by.